### PR TITLE
feat: adding new headers in scanners-user-agents.data

### PR DESCRIPTION
**Pull request by megalucio**

### CODE REVIEW

File: rules/scanners-user-agents.data [hunk 1]
```diff
@@ -215,3 +215,6 @@ Detectify
 # ZGrab scanner (Mozilla/5.0 zgrab/0.x)
 # https://zmap.io
 zgrab
+# Cyberscan
+# https://cyberscan.io
+zgrab
```
Comment (franbuehler) [inline rules/scanners-user-agents.data:220]: What is the user-agent name of the Cyberscan scanner?
You added zgrab here, but I don't think it's zgrab, right? This seems to be a copy paste from the entry above.

**Comment by franbuehler** [comment]
Thank you for your contribution! This is very welcome.
Could you please change the base of your PR to v4.0/dev instead of v3.3/master?
And I also added a comment to the file scanners-user-agents.data.

**Comment by dune73** [comment]
I'm not convinced we want this in 4.0.

We stripped down the allegedly malicious user agents until the bare minimum and this one is an edge case. Blocking cyberscan.io on the WAF because of the UA header might give you a false sense of security the is blown away by a real attacker using a different UA string.

**Comment by franbuehler** [comment]
Oh, yes, good point. So do we better close this PR unmerged?

**Comment by dune73** [comment]
Nothing wrong with keeping it open for some time and wait for other opinions.

**Comment by dune73** [comment]
I realize the PR actually aims at 3.3, so no 4.0 problem, but for 3.3 we no longer expand the coverage (-> release policy). So I'm afraid we really won't merge this.

**Pull request closed without merging**